.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Good job creating a `.gitignore` file , 👏 , however as required you should also ignore the `test.md` file 👍 .Kindly update this file to meet the requirement